allure-report-data/src/test/java/ru/yandex/qatools/allure/data/AllureReportGeneratorTest.java

```diff
@@ -112,15 +108,15 @@ public void shouldGenerateWithoutFailures() throws Exception {
         assumeTrue("Output directory must be empty ", listBefore != null && listBefore.length == 0);
         generator.generate(outputDirectory);
 
-        File dataDirectory = new File(outputDirectory, AllureReportGenerator.DATA_DIRECTORY_NAME);
+        File dataDirectory = new File(outputDirectory, ReportWriter.DATA_DIRECTORY_NAME);
         assertTrue("Data directory should be created", dataDirectory.exists());
 
-        assertThat(dataDirectory, contains(XUnitPlugin.XUNIT_JSON));
-        assertThat(dataDirectory, contains(TimelinePlugin.TIMELINE_JSON));
-        assertThat(dataDirectory, contains(BehaviorsPlugin.BEHAVIORS_JSON));
-        assertThat(dataDirectory, contains(DefectsPlugin.DEFECTS_JSON));
-        assertThat(dataDirectory, contains(EnvironmentPlugin.ENVIRONMENT_JSON));
-        assertThat(dataDirectory, contains(GraphPlugin.GRAPH_JSON));
+        assertThat(dataDirectory, contains("xunit.json"));
```

Comment:
I know u ask it =) I remove constants. Now plugin name specified in `@Plugin.Name` annotation, so constants is redundant